applications/Games/chess/manifest.yml

```diff
@@ -0,0 +1,16 @@
+sourcecode:
+  type: git
+  location:
+    origin: https://github.com/xtruan/flipper-chess.git
+    commit_sha: 78e21aab8e7fa5ed68c1499380e6963a6d476d0c
```

Comment:
This version is not buildable with ufbt - please merge https://github.com/xtruan/flipper-chess/pull/1

Reply:
Merged, thank you!